plugin/web/components/layout/WdLayout.tsx

```diff
@@ -0,0 +1,65 @@
+import { Layout, MenuProps } from "antd";
+import { MenuInfo } from "rc-menu/lib/interface";
+import React,{ memo, useMemo, useState } from "react";
+
+import "../../../node_modules/antd/dist/antd.less";
+import "../../styles/style.less";
```

Comment:
Same with this file as I said in another comment: I think you just need to import this in the main.tsx file and then good to go. Don't need to import in many components